### DISCUSSION TITLE
iOS & macOS Student

### DISCUSSION THREAD

**Discussion by sayan-lakhoua** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1188] [Q&A]
Thank you.

**Comment by MatthiasKerbl** [comment]
Hi @Sla1708,

I am glad to hear you like AlDente and appreciate the kind words.
We are already offering a student discount and you can get it at our partner [StudentAppCentre](https://www.studentappcentre.com/App/AlDente).

Best,
Matthias
